src/rgw/rgw_object_expirer.cc

```diff
@@ -89,7 +89,11 @@ int main(const int argc, const char **argv)
 
   const DoutPrefix dp(cct.get(), dout_subsys, "rgw object expirer: ");
   DriverManager::Config cfg;
+#ifdef WITH_RADOSGW_RADOS
   cfg.store_name = "rados";
+#else
```

Comment:
Probably "#elif defined(WITH_RADOSGW_POSIX)" ?  Maybe with a #error if neither is defined?